libpod/events.go

```diff
@@ -191,6 +191,16 @@ func (v *Volume) newVolumeEvent(status events.Status) {
 	}
 }
 
+// newSecretEvent creates a new event for a libpod secret
+func (r *Runtime) NewSecretEvent(status events.Status, secretID string) {
```

Comment:
I'm a little iffy on having this public, but since secrets don't live in Libpod I don't see another way

Reply:
Yeah - and there is the same thing done already for NewNetworkEvent.